doc/example_startup_scripts/linux_matlab.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+NRNML_LIBMEXPATH='"<..matlabroot..>/bin/glnxa64/"'
+NRNML_LIBNRNPATH='"<..neuron-directory..>/lib/"'
+NRNML_INTERFACEPATH='"<..matlabneuroninterface..>/neuron/"'
+
+export LD_LIBRARY_PATH=$NRNML_LIBMEXPATH:$NRNML_LIBNRNPATH:$NRNML_INTERFACEPATH:$LD_LIBRARY_PATH
```

Comment:
We also need to update PATH, so nrnivmodl executable can be found, and HOC_LIBRARY_PATH, so the .hoc files delivered with neuron can be found.